include/SDL_pixels.h

```diff
@@ -627,7 +627,35 @@ extern DECLSPEC void SDLCALL SDL_GetRGBA(Uint32 pixel,
  */
 extern DECLSPEC void SDLCALL SDL_CalculateGammaRamp(float gamma, Uint16 * ramp);
 
+/**
+ * Read one pixel's worth of data for a given pixel format.
+ *
+ * \param data a pointer to the pixel's data
+ * \param format an SDL_PixelFormat structure describing the format of the pixel
+ * \returns a pixel value suitable for passing to SDL_GetRGB or SDL_GetRGBA
+ *
+ * \since This function is available since SDL 2.0.18
+ *
+ * \sa SDL_WritePixelValue
+ * \sa SDL_GetRGB
+ * \sa SDL_GetRGBA
+ */
+extern DECLSPEC Uint32 SDLCALL SDL_ReadPixelValue(void *data, SDL_PixelFormat *format);
 
+/**
+ * Write one pixel's worth of data for a given pixel format.
+ *
+ * \param data a pointer to the memory where the pixel data is to be written
+ * \param format an SDL_PixelFormat structure describing the format of the pixel
+ * \param value a pixel value in the format you'd get from SDL_MapRGB or SDL_MapRGBA
+ *
+ * \since This function is available since SDL 2.0.18
+ *
+ * \sa SDL_ReadPixelValue
+ * \sa SDL_MapRGB
+ * \sa SDL_MapRGBA
+ */
+extern DECLSPEC void SDLCALL SDL_WritePixelValue(void *data, SDL_PixelFormat *format, Uint32 value);
```

Comment:
Minor detail, can you add a blank line here?

Reply:
I haven't got the time to make any of the more significant changes here right at the moment, but I have at least added the blank line. :-)